services/movie.py

```diff
@@ -1,11 +1,12 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Ensure that `genres_ids` and `actors_ids` are handled properly when they are `None`. Although the current implementation checks for their truthiness, it might be beneficial to explicitly handle `None` to avoid potential errors.